.github/workflows/build.yml

```diff
@@ -248,17 +248,20 @@ jobs:
             --mount "type=bind,src=$PWD/signed,dst=/doi-build/signed"
 
             # https://explore.ggcr.dev/?repo=docker/image-signer-verifier
-            docker/image-signer-verifier:0.3.3@sha256:a5351e6495596429bacea85fbf8f41a77ce7237c26c74fd7c3b94c3e6d409c82
+            "docker/image-signer-verifier:$BASHBREW_IMAGE_SIGNER"
 
             sign
 
             --envelope-style oci-content-descriptor
 
-            --aws_region "$AWS_KMS_REGION"
-            --aws_arn "awskms:///$AWS_KMS_KEY_ARN"
+            --kms="AWS"
+            --kms-region="$AWS_KMS_REGION"
+            --kms-key-ref="$AWS_KMS_KEY_ARN"
 
-            --input oci:///doi-build/unsigned
-            --output oci:///doi-build/signed
+            --input=oci:///doi-build/unsigned
+            --output=oci:///doi-build/signed
+            --referrers-dest="oisupport/staging-$BASHBREW_ARCH-$BUILD_ID"
```

Comment:
@tianon I am not sure if I can `BASHBREW_STAGING_TEMPLATE` here?

Reply:
I believe we were planning on having all build attestations going into the same repo e.g. `oisupport/referrers`

Reply:
aaah

Reply:
Thinking about the supply chain aspects / PoLP, we should threat model a little bit whether it might be problematic to have a compromised node that has write access to a shared repository like this. :thinking:

From that perspective, our "threat boundary" has traditionally been per-architecture since that roughly matches the risk, so it might actually make sense to do `ocisupport/referrers-ARCH` after all, but that _does_ make it hard/annoying to find the correct artifacts from a given digest in `library/xxx` :thinking:

Reply:
A repo per-architecture seems possible. Although, it will add some complexity to verification clients.  

Clients could dynamically set the image attestation repo by convention such as  `oisupport/referrers-$BASHBREW_ARCH` when verifying DOI. 

This could be asking a lot. I took a quick look into `cosign` and it isn't immediately clear the extent of support for this. I did find [specifying registry](https://github.com/sigstore/docs/blob/3a49fd9ecb8e8814e9eeebfaab114ee196f41260/content/en/system_config/registry_support.md?plain=1#L62), so there is some support for setting separate repos for artifact storage.

Reply:
Did some digging and stumbled on [something similar I added three years ago](https://github.com/sigstore/cosign/blame/main/cmd/cosign/cli/options/reference.go#L23) 😄  glad to see that stuck around!

We could do something with a similar argument but would have to pass it in with every call of [cosign verify-attestation](https://github.com/sigstore/docs/blob/main/content/en/verifying/attestation.md). Unless we did something clever in the TUF policy mapping file 🤷

Reply:
Yeah, we'd have to do something like we do for `library/`, I think (have something that trolls through known digests on `oisupport/referrers-*` and copies/merges them into `oisupport/referrers`).

The flip side is what's the harm if a compromised or misbehaving server deletes or uploads incorrect/invalid referrers?  That's what the signature checks are there to verify?  How do we recover from such an event?

Reply:
On the plus side, there are a fixed number of digests we'd need to look up tags for in each arch-specific repository _and_ they're all recorded in `builds.json`, so this _is_ tenable, but it's a lot of Hub requests to synchronize all that data. :disappointed:

Reply:
> The flip side is what's the harm if a compromised or misbehaving server deletes or uploads incorrect/invalid referrers? That's what the signature checks are there to verify? How do we recover from such an event?

Yes, the signatures would prevent spoofed attestations. If one were to get write access there would be a real DoS threat (deleting attestations to cause verification failures, bloating the referrers response, etc.).

Reply:
So is the agreement here that we are doing `ocisupport/referrers-ARCH`?

Reply:
No, all referrers will be pushed to `oisupport/referrers`.